app/code/Magento/Config/Model/Config/Backend/Email/Address.php

```diff
@@ -20,7 +20,8 @@ class Address extends \Magento\Framework\App\Config\Value
     public function beforeSave()
     {
         $value = $this->getValue();
-        if (!\Zend_Validate::is($value, 'EmailAddress')) {
+		$validator = new \Zend\Validator\EmailAddress();
+        if(!$validator->isValid($value)){
```

Comment:
Please fix formatting in all changed lines. 4 spaces must be used as indentation, there should be a space after `if` etc.

Reply:
Great 👍 Besides formatting changes look good to me.

Please also fix `\Magento\Newsletter\Controller\Subscriber\NewAction::validateEmailFormat`.